docs/how-to-contribute-to-apache-carbondata.md

```diff
@@ -7,14 +7,16 @@
     the License.  You may obtain a copy of the License at
 
       http://www.apache.org/licenses/LICENSE-2.0
-
+    
     Unless required by applicable law or agreed to in writing, software 
     distributed under the License is distributed on an "AS IS" BASIS, 
     WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
     See the License for the specific language governing permissions and 
     limitations under the License.
 -->
 
+[toc]
```

Comment:
Can you explain ,why add [toc] at here ?

Reply:
Try to add a **table of content** at the beginning of the doc, because It is a quite long documents and will be frequently queried for some specific part. It seems that it doesn't work in GitHub (while in my own interface it works well). 
I will fix it.